issue-pr-templates/bin/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Questions:
+  https://groups.google.com/forum/#!forum/loopbackjs
+  https://gitter.im/strongloop/loopback
+Immediate support:
+  https://strongloop.com/api-connect-faqs/
+  https://strongloop.com/node-js/subscription-plans/
+-->
+
+# Description/Steps to reproduce
+
+<!--
+If feature: A description of the feature
+If bug: Steps to reproduce
+-->
```

Comment:
What I meant was we should be more explicit in telling people up front that we need the sandbox from them for reproducing the issue or we won't look at it.